easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-CrayGNU-2016.11-Python-3.5.2.eb

```diff
@@ -0,0 +1,69 @@
+# @author: Guilherme Peretti-Pezzi (CSCS)
+
+easyblock = 'CMakeMake'
+
+name = 'OpenCV'
+version = '3.1.0'
+versionsuffix = '-Python-%(pyver)s'
+pyver = '3.5.2'
+
+homepage = 'http://opencv.org/'
+description = """OpenCV (Open Source Computer Vision Library) is an open source computer vision
+ and machine learning software library. OpenCV was built to provide
+ a common infrastructure for computer vision applications and to accelerate
+ the use of machine perception in the commercial products."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.11'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+source_urls = ['https://github.com/opencv/opencv/archive/']
+sources = ['%(version)s.zip']
+
+builddependencies = [
+    ('CMake', '3.6.0', '', True),
```

Comment:
Why not using /apps/common/UES/jenkins/SLES12/easybuild/modules/all/CMake/3.6.2 ?